ui/v2.5/src/index.scss

```diff
@@ -56,40 +80,84 @@ code {
 
 @media (min-width: 576px) {
   .zoom-0 {
-    width: 15rem;
+    width: 240px;
+
+    & .scene-card-video {
+      max-height: 180px;
+    }
+    & .previewable.portrait {
+      height: 180px;
+    }
   }
 
   .zoom-1 {
-    width: 20rem;
+    width: 320px;
+
+    & .scene-card-video {
+      max-height: 240px;
+    }
+    & .previewable.portrait {
+      height: 240px;
+    }
   }
 
   .zoom-2 {
-    width: 30rem;
+    width: 480px;
+
+    & .scene-card-video {
+      max-height: 360px;
+    }
+    & .previewable.portrait {
+      height: 360px;
+    }
   }
 
   .zoom-3 {
-    width: 40rem;
+    width: 640px;
+
+    & .scene-card-video {
+      max-height: 480px;
+    }
+    & .portrait {
+      height: 480px;
+    }
   }
 }
 
 .scene-card-video {
   height: auto;
   width: 100%;
+}
 
-  .zoom-0 {
-    height: 11.25rem;
+/* this is a bit of a hack, because we can't supply direct class names
+   to the react-select controls */
```

Comment:
Yeah, react-select is quite tedious to style. You can supply styles to it, but it has to be set on each individual part of the select and leads to a ton of boilerplate. This seems like a better approach tbh.